selfapi.js

```diff
@@ -367,6 +374,24 @@ API.prototype = {
     });
   },
 
+  // Build index routes.
+  buildIndexRoutes: function (basePath) {
+      var fullPath = normalizePath(this.path, basePath) || '/';
+      this.get({
+          title: 'Index',
+          handler: (request, response) => {
+              var routes = {};
+              Object.keys(this.children).forEach(child => {
+                  if (child == "null") {
+                      return;
+                  }
+                  routes[child.replace(/^\//, '')] = nodepath.join(fullPath, child);
```

Comment:
Yes, some exporters like [exportAllTests](https://github.com/JanitorTechnology/selfapi/blob/6cad7814e2bbfd4f892e25dc40f3237ca5b2a6da/selfapi.js#L124) have to produce absolute URLs, and they do this by accepting a `baseUrl` argument (e.g. `http://localhost:9000`, like [here](https://github.com/JanitorTechnology/janitor/blob/64924553f24efec7a05adf10716431a7034ccee1/tests/tests.js#L151)).